datadog/data_source_datadog_ip_ranges.go

```diff
@@ -48,6 +50,11 @@ func dataSourceDatadogIpRanges() *schema.Resource {
 				Computed:    true,
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
+			"synthetics_ipv4_by_location": {
+				Description: "List of IPv4 prefixes by location.",
```

Comment:
So that it winds up in documentation, do you think it'd be useful to mention how the list can be split here? (Not sure how much wording is needed to show that)

Reply:
Yeah I think that'll be useful, thanks for the suggestion!